### PR TITLE
DOCS-6701 OTel-to-DD metrics mapping guide

### PR DESCRIPTION
**Pull request by kayayarai**
### What does this PR do? What is the motivation?
New guide for OTel metrics mapping to Datadog 

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+The following diagram shows the process of mapping the metrics from OpenTelemetry into metrics that Datadog uses:
+
+{{< img src="opentelemetry/guide/metrics_mapping/mapping_process.png" alt="The decision process for mapping OpenTelemetry metric names to Datadog metric names. If an OTel metric is not used by any Datadog product, or if its semantics are the same as Datadog's, it is sent as-is to Datadog. Otherwise, a Datadog-style metric is created from the OTel metric and sent to Datadog." style="width:100%;" >}}
+
+## Use of the `otel` prefix
+
+To differentiate the metrics captured by the `hostmetrics` receiver from Datadog Agent, we add a prefix, `otel`, for metrics collected by the collector. If a metric name starts with `system.` or `process.`, `otel.` is prepended to the metric name. Monitoring the same infrastructure artifact using both Agent and Collector is not recommended. 
+
+<div class="alert alert-info">Datadog is evaluating how we can improve the OTLP metric experience, including potentially deprecating this <code>otel</code> prefix. If you have feedback related to this, please reach out your account team to provide your input.</div>
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:25]:
```suggestion
<div class="alert alert-info">Datadog is evaluating ways to improve the OTLP metric experience, including potentially deprecating this <code>otel</code> prefix. If you have feedback related to this, reach out your account team to provide your input.</div>
```

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+The metrics, mapped to Datadog metrics, are used in the following views:
+- [Infrastructure Host Map][5]
+- [Infrastructure List][6]
+- [Host default dashboards][7]
+- [APM Trace view Host info][8]
+
+**Note**: To correlate trace and host metrics, configure [Universal Service Monitoring attributes][3] for each service, and set the `host.name` resource attribute to the corresponding underlying host for both service and collector instances. 
+
+The following table shows what Datadog host metric names are associated with corresponding OpenTelemetry host metric names, and, if applicable, what math is applied to the OTel host metric to transform it to Datadog units during the mapping.
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:39]:
```suggestion
The following table shows which Datadog host metric names are associated with corresponding OpenTelemetry host metric names, and, if applicable, what math is applied to the OTel host metric to transform it to Datadog units during the mapping.
```

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+| Datadog metric name   | OTel metric name      | Metric description         | Transform done on OTel metric      |
+|-----------------------|-----------------------|----------------------------|--------------------------|
+| `system.load.1`         | `system.cpu.load_average.1m`     | The average system load over one minute. (Linux only)       |         |
+| `system.load.5`         | `system.cpu.load_average.5m`        | The average system load over five minutes. (Linux only)   | |
+| `system.load.15`        | `system.cpu.load_average.15m`     | The average system load over 15 minutes. (Linux only)         |            |
+| `system.cpu.idle`       | `system.cpu.utilization` <br>Attribute Filter state: `idle`    | Fraction of time the CPU spent in an idle state. Shown as percent    | Multiplied by 100     |
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:46]:
```suggestion
| `system.cpu.idle`       | `system.cpu.utilization` <br>Attribute Filter state: `idle`    | Fraction of time the CPU spent in an idle state. Shown as percent.   | Multiplied by 100     |
```

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+| `system.cpu.user`       | `system.cpu.utilization` <br>Attribute Filter state: `user`    | Fraction of time the CPU spent running user space processes. Shown as percent   | Multiplied by 100     |
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:47]:
```suggestion
| `system.cpu.user`       | `system.cpu.utilization` <br>Attribute Filter state: `user`    | Fraction of time the CPU spent running user space processes. Shown as percent.  | Multiplied by 100     |
```

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+| `system.cpu.system`     | `system.cpu.utilization` <br>Attribute Filter state: `system`  | Fraction of time the CPU spent running the kernel.    | Multiplied by 100    |
+| `system.cpu.iowait`     | `system.cpu.utilization` <br>Attribute Filter state: `wait`    | The percent of time the CPU spent waiting for IO operations to complete.         | Multiplied by 100       |
+| `system.cpu.stolen`     | `system.cpu.utilization` <br>Attribute Filter state: `steal`     | The percent of time the virtual CPU spent waiting for the hypervisor to service another virtual CPU. Only applies to virtual machines. Shown as percent | Multiplied by 100   |
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:50]:
```suggestion
| `system.cpu.stolen`     | `system.cpu.utilization` <br>Attribute Filter state: `steal`     | The percent of time the virtual CPU spent waiting for the hypervisor to service another virtual CPU. Only applies to virtual machines. Shown as percent.| Multiplied by 100   |
```

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+| `system.mem.total`      | `system.memory.usage`     | The total amount of physical RAM in bytes.   | Converted to MB (divided by 2^20) |
+| `system.mem.usable`     | `system.memory.usage` <br>Attributes Filter state: `(free, cached, buffered)` | Value of `MemAvailable` from `/proc/meminfo` if present, but falls back to adding `free + buffered + cached memory` if not. In bytes.    | Converted to MB (divided by 2^20) |
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:52]:
```suggestion
| `system.mem.usable`     | `system.memory.usage` <br>Attributes Filter state: `(free, cached, buffered)` | Value of `MemAvailable` from `/proc/meminfo` if present. If not present, falls back to adding `free + buffered + cached memory`. In bytes.    | Converted to MB (divided by 2^20) |
```

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+| `system.net.bytes_rcvd` | `system.network.io` <br>Attribute Filter direction: `receive`   | The number of bytes received on a device per second.       |         |
+| `system.net.bytes_sent` | `system.network.io` <br>Attribute Filter direction: `transmit`   | The number of bytes sent from a device per second.           |       |
+| `system.swap.free`      | `system.paging.usage` <br>Attribute Filter state: `free`    | The amount of free swap space, in bytes      | Converted to MB (divided by 2^20) |
+| `system.swap.used`      | `system.paging.usage` <br>Attribute Filter state: `used`     | The amount of swap space in use, in bytes   | Converted to MB (divided by 2^20) |
```
Comment (estherk15) [inline content/en/opentelemetry/guide/metrics_mapping.md:56]:
```suggestion
| `system.swap.used`      | `system.paging.usage` <br>Attribute Filter state: `used`     | The amount of swap space in use, in bytes.  | Converted to MB (divided by 2^20) |
```

**Review by estherk15 — APPROVED** [review]
Looks good, left a few optional suggestions. Approved!

File: content/en/opentelemetry/guide/metrics_mapping.md [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+
+[1]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/hostmetricsreceiver
+[2]: /opentelemetry/otel_collector_datadog_exporter/
+[3]: /universal_service_monitoring/setup/
+[4]: /opentelemetry/guide/semantic_mapping/
+[5]: https://app.datadoghq.com/infrastructure/map?fillby=avg%3Acpuutilization&groupby=availability-zone
+[6]: https://app.datadoghq.com/infrastructure
+[7]: /opentelemetry/otel_collector_datadog_exporter/?tab=onahost#out-of-the-box-dashboards
+[8]: /tracing/trace_explorer/trace_view/?tab=hostinfo
+[9]: https://app.datadoghq.com/dashboard/lists?q=containers%20-%20overview
```
Comment (kayayarai) [inline content/en/opentelemetry/guide/metrics_mapping.md:110]:
```suggestion
[9]: /opentelemetry/otel_collector_datadog_exporter/?tab=onahost#containers-overview-dashboard
```

**Pull request merged**